src/style.css

```diff
@@ -0,0 +1,122 @@
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.page__product-cards {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
you don't reeally need this flex here
```suggestion
```

Reply:
I understand. but if it no matter i want to use flex.